File: mycollab-services-ondemand/src/main/scala/com/esofthead/mycollab/module/billing/esb/DeleteAccountEvent.scala
package com.esofthead.mycollab.module.billing.esb

import com.esofthead.mycollab.common.domain.CustomerFeedbackWithBLOBs

/**
  * @author MyCollab Ltd
  * @since 5.1.0
  */
class DeleteAccountEvent(val accountId: Integer, val feedback: CustomerFeedbackWithBLOBs) {}

File: mycollab-services-ondemand/src/main/java/com/mycollab/ondemand/module/billing/esb/DeleteAccountEvent.kt
package com.mycollab.ondemand.module.billing.esb

import com.mycollab.common.domain.CustomerFeedbackWithBLOBs

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
class DeleteAccountEvent(val accountId: Int, val feedback: CustomerFeedbackWithBLOBs?)